src/storage/src/keyspace.rs

```diff
@@ -166,8 +168,42 @@ impl<S: StateStore> Keyspace<S> {
         self.store.iter(range, epoch).await
     }
 
+    pub async fn iter_strip_prefix(
```

Comment:
I'm considering whether to name `xx_strip_prefix` directly to `xx` so that the others won't misuse it anymore.